modules/capi-yaml-infrastructure-gcp.adoc

```diff
@@ -18,7 +18,7 @@ spec:
   network:
     name: <cluster_name>-network <2>
   project: <project> <3>
-  region: <region> <4>
+  region: <aws_region> <4>
```

Comment:
Does this region change apply to this page, since it's a GCP page?

Reply:
Hello @aldiazRH,
    Thanks for catching this issue. I reverted the change as it should not be included IMO. ML